MangoPay/KycDocument.php

```diff
@@ -3,30 +3,59 @@
 namespace MangoPay;
 
 /**
- * KYC document entity
+ * KYC document entity.
  */
 class KycDocument extends Libraries\Document
 {
     /**
+     * The type of the document for the user verification.
+     *
      * @var string
+     *
      * @see \MangoPay\KycDocumentType
+     * @see https://mangopay.com/docs/endpoints/kyc-documents#kyc-document-object
      */
     public $Type;
 
     /**
+     * The label of KYC document type.
+     *
      * @var string
-     * @see \MangoPay\KycDocumentStatus
+     * @phpstan-var \MangoPay\KycDocumentStatus::*
+     */
+    public $TypeLabel;
+
+    /**
+     * The status of the KYC document.
+     *
+     * @var string
+     *
+     * @see KycDocumentStatus
      */
     public $Status;
 
     /**
+     * The label of KYC document status.
+     *
+     * @var string
+     */
+    public $StatusLabel;
```

Comment:
Why did you add this property? I don't see it in the docs :D (correct me if I'm wrong)

Reply:
This property was identified in both the documentation and PHP SDK. I verified its existence through debug logs and implemented it accordingly.